src/components/Pages/Privacy/Privacy.vue

```diff
@@ -1,54 +1,839 @@
+            </li>
+            <li>
+              <h4>
+                In particular right to object and right to withdraw consent
+              </h4>
+              <ul>
+                <li>
+                  <b>Art. 21 GDPR - Right to object:</b> You have the right to object
+                  at any time, for reasons arising from your particular
+                  situation, to the processing of personal data concerning you
+                  which is necessary on the basis of a legitimate interest on
+                  our part or in order to carry out a task in the public
+                  interest, or which is carried out in the exercise of official
+                  authority.
+                </li>
+              </ul>
+              <br />
+              <p>
+                If you object, we will no longer process your personal data
+                unless we can prove compelling legitimate grounds for processing
+                that override your interests, rights and freedoms, or unless the
+                processing serves to assert, exercise or defend legal claims.
+              </p>
+              <p>
+                If we process your personal data for direct marketing purposes,
+                you have the right to object to the processing at any time. If
+                you object to processing for direct marketing purposes, we will
+                no longer process your personal data for these purposes.
+              </p>
+              <p>
+                In order to exercise your right of objection, you can, for
+                example, send us an email to the email address mentioned above.
+              </p>
+              <ul>
+                <li>
+                  <b>Withdrawal of consent:</b> If you have given us your informed
+                  consent, you have the right to withdraw your consent at any
+                  time. In this case, all data processing that we have carried
+                  out until your withdrawal remains lawful.
+                </li>
+              </ul>
+              <br />
+            </li>
+            <li>
+              <h4>Obligation to provide data</h4>
+              <p>
+                You have no contractual or legal obligation to provide us with
+                personal data. However, we then might not be able to offer you
+                the requested services.
+              </p>
+            </li>
+            <li>
+              <h4>
+                The existence of automated decision-making (including profiling)
+              </h4>
+
+              <p>
+                We will not make you subject to any automated decision-making,
+                including profiling in accordance with Art. 22 para. 1 and 4
+                GDPR, which has legal effects on you or affects you.
+              </p>
+            </li>
+            <li>
+              <h4>Internet-specific data processing and cookies</h4>
+              <h5>Cookies</h5>
```

Comment:
Could you make `Cookies` and `Google reCaptcha` a bit bigger? <h5> makes it suddenly tiny.
May be scale it up a bit?
```
h5 {
  font-size: 120%;
}
```